examples/tpm2-ekcert/main.go

```diff
@@ -17,13 +20,41 @@ var (
 	// Default value is defined in section 7.8, "NV Memory" of the latest version pdf on:
 	// https://trustedcomputinggroup.org/resource/tcg-tpm-v2-0-provisioning-guidance/
 	certIndex = flag.Uint("cert-index", 0x01C00002, "NVRAM index of the certificate file")
+	tmplIndex = flag.Uint("template-index", 0, "NVRAM index of the EK template; if empty, default RSA EK template is used")
```

Comment:
One more minor nit: "empty" doesn't make sense to me in the context of an integer flag. I was going to suggest "if not set", but that's not true either, according to the code. It looks like it should say "if non-zero"

Reply:
Done